src/ios/ELCImagePicker/ELCAssetCell.m

```diff
@@ -81,12 +104,15 @@ - (void)cellTapped:(UITapGestureRecognizer *)tapRecognizer
     
 	CGRect frame = CGRectMake(startX, 2, 75, 75);
 	
-	for (int i = 0; i < [_rowAssets count]; ++i) {
+    assert([_checkIconArray count] == [_circleIconArray count]);
```

Comment:
Asset crashes the app when it fails, we certainly don't want that behavior. either use NSAssert so you can include a nice exception message or use an if statement. 🌮 [ here](https://stackoverflow.com/a/6620709/2603945)